app/src/pages/NoteList/components/NotesLayout/styled.ts

```diff
@@ -0,0 +1,145 @@
+import styled from 'styled-components';
+import { Box } from '@mui/material';
+
+import { COLORS } from 'config/colors';
+import { StyledButton } from 'components/FilterNotes/styled';
+
+import { INotesButton, IStyleNote } from './types';
+
+const StyledNotesLayout = styled(Box)({
+  display: 'block',
+  flexWrap: 'wrap',
+  justifyContent: 'center',
+  alignItems: 'center',
+  marginTop: '0.5rem',
+  height: '71vh',
+  overflowY: 'scroll',
+  '&>div': {
+    '&>div': {
+      overflow: 'hidden !important',
```

Comment:
remove important

Reply:
if we can discuss how to fix it, I've tried to remove important, without it i couldn't set overflow: hidden